examples/hscy3/pipeline.yml

```diff
@@ -0,0 +1,183 @@
+    #- name: TXTwoPointTheoryReal # compute theory using CCL to save in sacc file and plot later
+    #- name: TXTwoPointPlotsTheory      # Make plots of 2pt correlations
+    #- name: TXSourceDiagnosticPlots    # Make a suite of diagnostic plots
+    #- name: TXLensDiagnosticPlots    # Make a suite of diagnostic plots
+    #- name: TXGammaTFieldCenters # Compute and plot gamma_t around center points
+    #  threads_per_process: 2
+    #- name: TXGammaTStars  # Compute and plot gamma_t around bright stars
+    #  threads_per_process: 2
+    #- name: TXGammaTRandoms      # Compute and plot gamma_t around randoms
+    #  threads_per_process: 2
+    #- name: TXGalaxyStarDensity  # Compute and plot the star-galaxy density cross-correlation
+    #- name: TXGalaxyStarShear    # Compute and plot the star-galaxy shear cross-correlation
+    #- name: TXPhotozPlotSource        # Plot the bin n(z)
+    #  classname: TXPhotozPlot
+    #- name: TXPhotozPlotLens        # Plot the bin n(z)
+    #  classname: TXPhotozPlot
+    #- name: TXConvergenceMaps    # Make convergence kappa maps from g1, g2 maps
+    #- name: TXConvergenceMapPlots # Plot the convergence map
+    #- name: TXMapCorrelations    # plot the correlations between systematics and data
+    #- name: TXApertureMass        # Compute aperture-mass statistics
+    #  threads_per_process: 2
+    #- name: TXTwoPointFourier    # Compute power spectra C_ell
+    # Disablig this since not yet synchronised with new Treecorr MPI
+    # - name: TXSelfCalibrationIA   # Self-calibration intrinsic alignments of galaxies
+    # Disabling these as they take too long for a quick test.
+    # - name: TXRealGaussianCovariance   # Compute covariance of real-space correlations
+    # - name: TXTwoPointFourier          # Compute power spectra C_ell
+    # - name: TXFourierNamasterCovariance # Compute the C_ell covariance
+    # - name: TXFourierTJPCovariance     # Compute the C_ell covariance with TJPCov 
+
+
+# modules and packages to import that have pipeline
+# stages defined in them
+modules: >
+    txpipe
+    rail.creation.degradation.grid_selection
+    rail.creation.engines.flowEngine
+    rail.estimation.algos.NZDir
+    rail.estimation.algos.bpz_lite
+
+# where to find any modules that are not in this repo,
+# and any other code we need.
+python_paths:
+    - submodules/WLMassMap/python/desc/
+
+# Where to put outputs
+output_dir: data/hsc-y3/outputs
+
+# How to run the pipeline: mini, parsl, or cwl
+launcher:
+    name: mini
+    interval: 1.0
+
+# Where to run the pipeline: cori-interactive, cori-batch, or local
+site:
+    name: local
+    max_threads: 2
+
+# configuration settings
+config: examples/desy3/config_Bmodes.yml
+
+# These are overall inputs to the whole pipeline, not generated within it
+inputs:
+    # See README for paths to download these files
+    shear_catalog     : /global/cfs/cdirs/lsst/groups/WL/projects/txpipe-sys-tests/hsc-y3/shear_catalog_desy3_unmasked_withfakez_v2.h5
```

Comment:
small error: des y3 files here

Reply:
This was an accidental push of an old yaml. This has been replaced with `hsc_pipeline.yml`